src/extension/webbtc/index.ts

```diff
@@ -0,0 +1,125 @@
+type RequestInvoiceArgs = {
+  amount?: string | number;
+  defaultAmount?: string | number;
+  minimumAmount?: string | number;
+  maximumAmount?: string | number;
+  defaultMemo?: string;
+};
+
+export default class WebBTCProvider {
```

Comment:
Is there a way to reuse this code from webln or move out and import it in both providers? Would like to avoid that we have to maintain this two times. Extend it in one, forget it in the other etc

Reply:
Done!